### ISSUE TITLE
client watch multiple key range will panic

### ISSUE THREAD

**Issue by SSebo** [luncj/etcd-rs#55]
panic position:
```
 thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', etcd-rs/src/watch/mod.rs:182:66
```

watch.tunnel.resp_receiver can not take again:
```
tunnel.resp_receiver.take()
```

**Comment by SSebo** [comment]
I made a PR  https://github.com/luncj/etcd-rs/pull/56

**Comment by luncj** [comment]
closed via #56
